src/QA/QA.Web/ViewModels/Validations/ReviewEditModelValidator.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Esfa.Recruit.Vacancies.Client.Application.Validation.Fluent;
+using FluentValidation;
+
+namespace Esfa.Recruit.Qa.Web.ViewModels.Validations
+{
+    public class ReviewEditModelValidator : AbstractValidator<ReviewEditModel>
```

Comment:
Is this a move away from the pattern we use in Employer?

Reply:
I can see the benefit of pushing Vacancy validations through to the client as we modify it over a series of pages. However for simpler entity interactions I think there is no benefit in adding this extra complexity (E.g. Overhead of marshalling entity validations to their VM counterparts).
So for VacancyReview I've thought adding view model validations and equivalent entity validations is the simpler solution for this ticket. Happy to change if needed.

Reply:
Yep I take your point. We will just have the downside of having to keep the 2 validators in sync.